ui/src/css/_common.scss

```diff
@@ -0,0 +1,80 @@
+.mdc-button {
+  text-transform: none;
```

Comment:
There's a couple categories of things going on:
* For the the text-transform part, yes, it's explicitly overriding a Material setting in order to match the specs we got from UX. Similarly, for the headline5 size, it's because the specs call for a text size that isn't a Material option. I'll bring these up with Travis and see if he wants to change it.
* For the card spacing-related stuff, it's because I was using Material classes where, now that I think about it, I should have made my own class (e.g., it's necessary to mess around with card padding to get the left-side background color to extend all the way to the edge, because Material doesn't have its own support for cards arranged like the repo cards). I'll fix that and ping this thread when that's done.
* Lastly, for the coloring: Material expects you to have two colors (a primary and secondary) and uses those in various places. Our colors are blue and white, which can be an issue because some places Material wants to use our secondary color are places where white is a bad choice (e.g., cursors). There's probably a better way to handle this so I'll try finding it (since I have to define two button types of my own anyway, maybe I can just tell Material gray is our secondary color).

Reply:
Thanks for explanation! Feel free to submit it as is for now and improve it later, but can you add comment for each of them with its intent (as you explained here)? It would be useful to know e.g., if it's an intentional divergence from Material Design or a workaround of a bug of Material Design library.

Reply:
Ok, added some comments and TODOs.